### ISSUE TITLE
Clean up branches - master only branch

### ISSUE THREAD

**Issue by jaredsburrows** [AzureAD/azure-activedirectory-library-for-android#807]
Can we reduce our branches down to `master` only?

The default branch _should_ be `master` but it's current `dev`. I would like to help contribute but having two development branches adds confusion and complexity. Do you all decide to merge with `master` on releases?

After switching the default branch to master, we can start publishing `snapshots` to `mavenCentral()` off of master that way developers can have their changes and test them right away. For example, if I make a PR contribution and my QA team would like to try out the contribution, they would have to clone this repository and manually compile the library themselves. I want to avoid this and have continuous integration and deployment off of `master`. What do you all think?

**Comment by jaredsburrows** [comment]
What do you think @heidijinxujia?

**Comment by weijjia** [comment]
Master is our release branch, no changes should go to master directly. Dev is the development branch, all the features, bug fixes should go to dev first. Only when we decide to do a release, we merge to master.

**Comment by jaredsburrows** [comment]
Alright. I guess we can close this. It just makes it hard to follow and develop. 

For the snapshots, you all should set that up through the CI since you already have maven central approval.
